src/App.tsx

```diff
@@ -2,15 +2,26 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { useState } from 'react';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+
+  const addMovie = (movie: Movie) => {
+    setMovies(currentMovie => ([
```

Comment:
```suggestion
    setMovies(currentMovies => ([
```
Actually it's an array of movies